lib/spack/spack/cmd/create.py

```diff
@@ -120,9 +122,12 @@ class PackageTemplate(BundlePackageTemplate):
     """Provides the default values to be used for the package file template"""
 
     base_class_name = "Package"
+    build_system_import = "import spack.build_systems.generic"
```

Comment:
```suggestion
    build_system_import = "\nimport spack.build_systems.generic"
```
So that the whitespace is consistent